modules/document_repository/php/NDB_Menu_Filter_document_repository.class.inc

```diff
@@ -77,9 +77,11 @@ class NDB_Menu_Filter_Document_Repository extends NDB_Menu_Filter
                           'v.Date_uploaded',
                          );
 
-        $openAccordion = $_GET['openAccordion'];
+        $openAccordion = isset($_GET['openAccordion']) ?
```

Comment:
Well, this is a pretty important fix, imo. So, I'll make the change now because I'm all for adopting php7.

We are moving to php7 before this release, right? Because this pull request needs to be in the release.